LonaStudio/Models/CSData.swift

```diff
@@ -229,7 +223,7 @@ enum CSData: Equatable, CustomDebugStringConvertible {
         case .Null:
             return NSNull()
         case .Bool(let value):
-            return NSNumber(booleanLiteral: value)
+            return NSNumber(value: value)
```

Comment:
I'm pretty sure this does actually need to be `booleanLiteral` or it won't turn into a bool when written as JSON.